foundation-extensions/mobile-core/platform-services.md

```diff
@@ -1,6 +1,122 @@
 # Platform Services
 
-Currently, the AEP SDK only provides the capability for overriding the Adobe-provided network stack (for all Adobe extensions).
+The Platform Services are provided by the Adobe Experience Platform Mobile SDKs as part of the Mobile Core extension. These services provide shared functionality throughout the SDK that can be shared by extensions. For example, services provide shared functionality for networking, data queuing, caching, and more.
+
+{% hint style="info" %}
+This feature is only available in [Android Core 1.8.0](https://aep-sdks.gitbook.io/docs/release-notes#android-core-1-8-0) or later and [iOS Core 3.0.0](https://aep-sdks.gitbook.io/docs/release-notes#adobe-experience-platform-ios-core-sdks) or later.
+{% endhint %}
+
+## Accessing Services
+
+The MobileCore extension provides a shared `ServicesProvider`, responsible for accessing the current set of provided services.
```

Comment:
I think we need to mention that the ServiceProvider (not ServicesProvider) is part of the AEPServices extension. Mainly because we need to mention that they need to import `AEPServices` whenever they are using the service provider.